### ISSUE TITLE
Can't access admin.php

### ISSUE THREAD

**Issue by BlackmanAlice** [impresspages/ImpressPages#681]
I am having problems accessing my admin page.Every time after I had put my login details and submited a blank screen apears with no code,sometimes even that login page does not appear.
I downloaded impresspages via softaculous

**Comment by maskas** [comment]
First, open config.php file and make sure showErrors setting is set to 1. That will show you an error that's happening when you see a blank screen.

**Comment by BlackmanAlice** [comment]
Thank you for the reply.
I have opened config.php and found that showErrors is set to 1 is there any other way to try?

**Comment by maskas** [comment]
Do you have an URL to have a look at? Do the rest of the website works as expected?

If you have any plugins, try to remove them to see if they are causing any problems.

**Comment by BlackmanAlice** [comment]
After a while now I have managed to log in but it seems I can only log in using my tablet and it gives me limited access as there are sections I cant access,still on my computers I can not access the admin.
I realised that the addthis plugin was not functioning properly so I removed it using the tablet,still the same problems remains since I can't access it using my computer.
here is the URL www.blackmanalice.co.za

**Comment by maskas** [comment]
Do you have JavaScript enabled on your computer?  As it looks it works fine for me. At least if I enter wrong credentials, I get an error as expected. So it looks working.

And by the way, which browser are you using? And which version?

**Comment by BlackmanAlice** [comment]
Yes JavaScript is enabled on my computer.I am using Google chrome Version 40.0.2214.93 m

**Comment by maskas** [comment]
By the way which one are you using?

http://www.blackmanalice.co.za/admin.php
ir
http://www.blackmanalice.co.za/admin

As it looks like you really have some problems with admin.php. It looks like it is just an empty file.

**Comment by BlackmanAlice** [comment]
On the tablet I used http://www.blackmanalice.co.za/admin.php as it is the one I have been using ever since I started using Impress pages.
Just now after I had seen your question I tried http://www.blackmanalice.co.za/admin and it worked just fine both on the desktop and laptop.
Thank you very much I wonder though what might have went wrong with http://www.blackmanalice.co.za/admin.php that it has become an empty file?
